mod_reforged/config/item_names.nut

```diff
@@ -1,3 +1,52 @@
+// Vanilla Adjustments
+::MSU.Array.removeValues(::Const.Strings.CrossbowNames, [
+	"Betrayer",		// moved over to dagger names
+]);
+::Const.Strings.CrossbowNames.extend([
+	"Scorpion",
+	"Ironsight",
+	"Windlass",
+]);
+
+::MSU.Array.removeValues(::Const.Strings.DaggerNames, [
+	"Estoc",		// We have our own actual Kriegsmesser item
+]);
```

Comment:
This seems like a mistake.

Reply:
Ah I see there's a dagger name Estoc in vanilla dagger names. Ok then the removal is good, but please update the comment to mention Estoc instead of Kriegsmesser.